src/sentry/options/defaults.py

```diff
@@ -362,6 +362,9 @@
 # All Relay options (statically authenticated Relays can be registered here)
 register("relay.static_auth", default={}, flags=FLAG_NOSTORE)
 
+# Tell Relay to stop extracting metrics from transaction payloads
+register("relay.drop-transaction-metrics", default=[])
```

Comment:
Do I need another PR in `getsentry` for this?

Reply:
you need a separate PR for introducing this option, which we then have to deploy separately and set the option in prod to the default value

Reply:
Kind of off topic, but could we fix this so that we can happily introduce an option like this but it is just going to return the default (here the empty list) until the database thing is done?